.gitlab/pipelines/stages/deploy/wait_kubecf.sh

```diff
@@ -5,10 +5,16 @@ set -o errexit -o nounset
 # shellcheck disable=SC1090
 source "$(bazel info workspace)/.gitlab/pipelines/config/config.sh"
 
+get_endpoint() {
+  bazel run @kubectl//:kubectl -- get endpoint kubecf-router-public \
+    --namespace "${KUBECF_NAMESPACE}" \
+    --output jsonpath='{.subsets[0].addresses[0].ip}' 2> /dev/null
+}
+
 # Wait for kubecf to start.
 wait_for_router() {
   local timeout="1800"
-  until curl -k --fail --head --silent "https://api.${SYSTEM_DOMAIN}/v2/info" || [[ "$timeout" == "0" ]]; do sleep 1; timeout=$((timeout - 1)); done
+  until [[ "$(get_endpoint)" != "" ]] || [[ "$timeout" == "0" ]]; do sleep 1; timeout=$((timeout - 1)); done
```

Comment:
```bash
local timeout
for (( timeout = 1800 ; timeout > 0 ; timeout -- )) ; do
  if [[ "$(get_endpoint)" != "" ]] ; then
    return 0
  fi
  sleep 1
done
return 1
```

But… is this as effective as waiting for the API endpoint? I mean, yes it's better in that it's _actually_ waiting for just the router now… But it's definitely no longer `Wait for kubecf to start`.

Reply:
We should come up with a better alternative that works when we use ClusterIP for the public router service.